src/resources/team/team.service.ts

```diff
@@ -26,5 +26,21 @@ export const createOne = async (
 	user.teamIds.push(teamIds)
 	await user.save()
 
-	return newTeam
+	return Promise.resolve(newTeam)
+}
+
+/**
+ * Get teams by user id
+ *
+ * @param userId
+ */
+export const getByUserId = async (userId: string): Promise<[TeamDocument]> => {
+	logger.debug(`Get teams by user id: %o`, userId)
+
+	let query = UserModel.findOne({_id: userId})
```

Comment:
could use findById, use const instead of let or you could chain query like this

`const data = await UserModel.findById(userId).populate('teams').lean().exec()`

Remember to throw error if user not found (data = null)